<element name="noise" required="+">
  <description>The properties of a sensor noise model.</description>

  <element name="type" type="string" default="gaussian" required="1">
    <description>
      The type of noise. Currently supported types are:
      "gaussian" (draw noise values independently for each measurement from a Gaussian distribution).
      "gaussian_quantized" ("gaussian" plus quantization of outputs (ie. rounding))
    </description>
  </element>
  <element name="mean" type="double" default="0.0" required="0">
    <description>For type "gaussian*", the mean of the Gaussian distribution from which noise values are drawn.</description>
  </element>
  <element name="stddev" type="double" default="0.0" required="0">
    <description>For type "gaussian*", the standard deviation of the Gaussian distribution from which noise values are drawn.</description>
  </element>
  <element name="bias_mean" type="double" default="0.0" required="0">
    <description>For type "gaussian*", the mean of the Gaussian distribution from which bias values are drawn.</description>
  </element>
  <element name="bias_stddev" type="double" default="0.0" required="0">
    <description>For type "gaussian*", the standard deviation of the Gaussian distribution from which bias values are drawn.</description>
  </element>
  <element name="precision" type="double" default="0.0" required="0">
    <description>
      For type "gaussian_quantized", the precision of output signals. A value
      of zero implies infinite precision / no quantization.
    </description>
  </element>

</element>
